packages/select/src/components/select/select.tsx

```diff
@@ -175,13 +175,16 @@ export class Select<T> extends AbstractPureComponent<SelectProps<T>, SelectState
 
         const input = (
             <InputGroup
+                aria-activedescendant={listProps.activeItemId}
```

Comment:
Why are we setting [`aria-activedescendant`](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Reference/Attributes/aria-activedescendant) on the filter input group? Shouldn't we be setting this on the outermost combobox element?

Here's the WAI [Select-Only Combobox Example](https://www.w3.org/WAI/ARIA/apg/patterns/combobox/examples/combobox-select-only) for reference:

<img width="2816" height="1676" alt="Screenshot 2025-08-13 at 12 14 27@2x" src="https://github.com/user-attachments/assets/50adf57c-10db-44e6-ab70-779b5f85e8ff" />